lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -128,6 +136,12 @@ std::unique_ptr<CompiledFunction> CPUBackend::compile(Function *F) const {
   return compileIR(std::move(IR));
 }
 
+std::unique_ptr<CompiledFunction>
+CPUBackend::compileWithoutConstants(Function *F) const {
+  auto IR = generateAndOptimizeIR(F, shouldShareBuffers());
+  return compileIRWithoutConstants(IR.get());
```

Comment:
Did we come to a conclusion about how `collectConstants` is going to work for the CPU backend yesterday?  Since `compileWithoutConstants` throws away the IR, we don't have anything to pass to `collectConstants`...

Reply:
I think we decided to make expose collect constants as a method on each compiled function, and keep the bundle_ private. We call collectConstants in CompileIR where we still have the IR.